apps/wallet/src/ui/app/pages/home/tokens/TokenIconLink.tsx

```diff
@@ -55,8 +54,8 @@ export function TokenIconLink({
             onClick={() => {
                 trackEvent('StakingFromHome');
             }}
-            className={cl(
-                !stakingEnabled && '!bg-gray-40',
+            className={cx(
```

Comment:
nit: let's use the conditional object functionality of `cx` so this is more maintainable

```
cx('flex rounded-2xl w-full p-3.75 justify-between no-underline bg-sui/10', {
 '!bg-gray-40': !stakingEnabled
})
```